machAeroTutorials/aero_gridRefinementStudy.rst

```diff
@@ -47,15 +47,39 @@ It is computed using the solutions from the L0 and L1 grids with the equation
 .. math::
     f_{h=0} = f_{L0} + \frac{f_{L0}-f_{L1}}{r^{\hat{p}} - 1}
 
-Grid refinement study with ADflow
----------------------------------
+Grid Refinement Study on Airfoils
+======================================
 
-1. Generate fine grid (L0) with :math:`N=2^n m + 1` nodes along each edge.
-2. Coarsen the L0 grid :math:`n-1` times using :code:`cgns_utils coarsen`.
-3. Use :code:`solveCL` or :code:`solveTrimCL` in ADflow to obtain :math:`C_D` for a given :math:`C_L`.
+There are two methods for performing grid refinement: 
+1) coarsening the volume mesh and 
+2) coarsening the surface mesh and extruding the family of surface meshes
+We discuss the pros and cons of each method and underlying theory;
+it is up to the user to choose the method.
+
+A few general notes first:
+   - With regards to aerodynamic shape optimization, being in the asymptotic regime is not always necessary since the objective function like drag is just offset by the truncation error.
```

Comment:
I don't think this is the best way to put it. You can always say the value is "just offset by the truncation error". That does not mean the offset is always the same throughout the optimization. Capturing the correct physical trends (as mentioned later) is the most important part, but being in the asymptotic range should still be a goal.